app/features/navbar/components/HelpButton.js

```diff
@@ -97,9 +76,20 @@ export default class HelpButton extends Component< *, State> {
                 onOpenChange = { this._onIconClick }
                 position = 'right bottom'
                 trigger = { <HelpIcon /> }>
-                <Item onActivate = { this._onTermsOptionClick }>Terms</Item>
-                <Item onActivate = { this._onPrivacyOptionClick }>Privacy</Item>
-                <Item onActivate = { this._onSendFeedback }>Send Feedback</Item>
+                <Group heading = 'Help'>
```

Comment:
Sorry I didn't at the UI previously.
Should we really add this? The UI looks a little bit weird to me.

Reply:
It's consistent with other UI I've seen using AtlasKit.

Reply:
@saghul 
Hmm. Yeah, it is. I just looked at stride and bitbucket.
Maybe it looks weird to me because it is centre-aligned. 
Totally your call. :smile: